apps/angular/signal-input/src/app/user.component.ts

```diff
@@ -19,23 +21,31 @@ const ageToCategory = (age: number): Category => {
   standalone: true,
   imports: [TitleCasePipe],
   template: `
-    {{ fullName | titlecase }} plays tennis in the {{ category }} category!!
+    {{ fullName() | titlecase }} plays tennis in the {{ category() }} category!!
   `,
   host: {
     class: 'text-xl text-green-800',
   },
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class UserComponent implements OnChanges {
-  @Input({ required: true }) name!: string;
-  @Input() lastName?: string;
-  @Input() age?: string;
+export class UserComponent {
+  name: InputSignal<string> = input.required<string>();
+  lastName: InputSignal<string | unknown> = input<string | unknown>();
```

Comment:
add a default value when the input is not required. it will avoid using the coalapsing operator later. 

Why did you use the unknonw type here?